utils.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const path = require('path');
+const logger = require('debug')('uni-config');
+
+const environment = process.env.NODE_ENV || 'development';
+const debug = environment === 'development';
+const sandbox = process.env.SANDBOX === 'true' || debug;
+const configDir = process.env.CONFIG_DIR || 'config';
+
+const base = { environment, debug, sandbox };
+
+logger('building config with %O', base);
+
+const getPath = (name) => path.resolve(`${__dirname}/../../${configDir}/${name}`);
```

Comment:
`process.cwd`?

Reply:
that _should_ work but i don't want to risk this change atm